### DISCUSSION TITLE
Forbidden Error: ExtensionPay Profile

### DISCUSSION THREAD

**Discussion by TyndaleH** [Glench/ExtPay#77] [General]
Hi - I was completing my account at Stripe and changed my mind about some things so I started over the process and created a different Stripe account to eventually link with ExtensionPay. 

Now when I try to access extensionpay.com it loads a blank page with "Forbidden". Any help getting my Extensionpay profile reset would be greatly appreciated, thanks!

**Comment by Glench** [comment]
You should be all set now.

**Reply by TyndaleH** [reply]
Great - Thank you for the quick help!

**Comment by RajatVaghani** [comment]
Hey I'm facing it too now!

**Reply by Glench** [reply]
fixed!

**Reply by RajatVaghani** [reply]
Thank you, that worked!

now a different issue though. 

how do I process payments from India?

<img width="1449" alt="Screenshot 2022-11-27 at 1 10 06 AM" src="https://user-images.githubusercontent.com/12733424/204106373-59bc14cf-848e-4d4f-9177-b06278ab13e0.png">

**Reply by Glench** [reply]
Yes, there are some things you need to know before signing up with Stripe in India. Please email glen@extensionpay.com and I can get you set up.

**Comment by Netflixyapp** [comment]
Hi, we started getting the same error "Forbidden" after closing the linked Stripe account (trying to link a new one).

**Reply by Glench** [reply]
This should have already been fixed for you.
